app/gui/src/model/suggestion_database/mock.rs

```diff
@@ -321,6 +322,94 @@ macro_rules! mock_suggestion_database {
     }
 }
 
+/// A helper macro for mocking entry's documentation.
+///
+/// ### [`DocSection::Paragrah`]
+/// ```
+/// # use enso_gui::doc_section;
+/// doc_section!("Some text.");
+/// ```
+///
+/// ### [`DocSection::Tag`]
+/// ```
+/// # use enso_gui::doc_section;
+/// doc_section!(@ "Tag name", "Tag body.");
+/// ```
+///
+/// ### [`DocSection::Keyed`]
+/// ```
+/// # use enso_gui::doc_section;
+/// doc_section!("Key" => "Value")
+/// ```
+///
+/// ### [`DocSection::Marked`]
+/// ```
+/// # use enso_gui::doc_section;
+/// doc_section!(! "Marked as important");
+/// doc_section!(! "Optional header", "Marked as important");
+/// doc_section!(? "Marked as info");
+/// doc_section!(? "Optional header", "Marked as info");
+/// doc_section!(> "Marked as example");
+/// doc_section!(> "Optional header", "Marked as example");
+/// ```
+#[macro_export]
+macro_rules! doc_section {
+    ($paragraph:expr) => {
+        engine_protocol::language_server::DocSection::Paragraph { body: $paragraph.into() }
```

Comment:
change `engine_protocol::language_server::DocSection` to `$crate` and re-export it from the crate. Alternatively, change it to `$crate::app::gui::suggestion_database::...`. Macros that are exported should always succeed even if `engine_protocol` was not used in cargo toml config.

Reply:
Fixed